test/run-tests

```diff
@@ -468,7 +468,11 @@ def choose_task(gh, repos, images):
             commands = get_comment_commands(gh, owner, repo, number)
 
             for image in images:
-                status = statuses.get("linux-system-roles-test/" + image["name"])
+                status_context = "%s/%s" % (
+                    config.get("test_harness_name", "linux-system-roles-test"),
```

Comment:
Also I think it could just be `name` in config.json. Since it is the config for test-harness there is no need to add it to the config option as well IMHO.